pom.xml

```diff
@@ -426,7 +426,7 @@
 
 					<!-- Third argument prevents Java from popping up lots of windows on
 						MacOS -->
```

Comment:
Well my own dev environment are two laptops with 32GB and 16GB respectively, but I like to have headroom for active development while running the tests. Honestly, I don't see a good reason why we should unnecessarily increase this requirement. Btw, even the Spark default driver memory setting is 1GB.